src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/FareProducts/RecentFareProducts/RecentFareContractComponent.tsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import {ThemeText} from '@atb/components/text';
-import {FareContractTexts, useTranslation} from '@atb/translations';
+import {useTranslation} from '@atb/translations';
 import RecentFareContractsTexts from '@atb/translations/screens/subscreens/RecentFareContractsTexts';
 import {RecentFareContract} from '../use-recent-fare-contracts';
 import {StyleSheet, useTheme} from '@atb/theme';
 import {Dimensions, TouchableOpacity, View} from 'react-native';
 import {getReferenceDataName} from '@atb/reference-data/utils';
-import {TransportationIcon} from '@atb/components/transportation-icon';
 import {ThemeIcon} from '@atb/components/theme-icon';
 import {ArrowRight} from '@atb/assets/svg/mono-icons/navigation';
 import {useFirestoreConfiguration} from '@atb/configuration/FirestoreConfigurationContext';
 import {FareProductTypeConfig} from '@atb/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/FareContracts/utils';
-import {TransportModeType} from '@atb/configuration/types';
 import {InfoChip} from '@atb/components/info-chip';
 import {InteractiveColor} from '@atb/theme/colors';
+import {
+  getTransportModeText,
+  TransportMode,
```

Comment:
Since the `TransportMode` component now is used more places, maybe it can be extracted to its own component?

Reply:
I moved it to src/components folder, let me know if that was not what you meant